doc/gazebo_simulation/gazebo_simulation.rst

```diff
@@ -0,0 +1,351 @@
+
+5. Configure gazebo_ros_control, transmissions and actuators
+------------------------------------------------------------
+
+This is necessary for the robot to move in Gazebo. ROS Control is a highly capable robot-agnostic stack, providing interfaces
+to control theoretically any type of robot. :code:`gazebo_ros_control` enables the ROS control to be used in Gazebo. 
+See `its document <http://gazebosim.org/tutorials/?tut=ros_control>`_ for full details.
+
+Add the following to the newly created :code:`panda.gazebo.xacro` file:
+
+.. code-block:: xml
+
+    <gazebo>
+        <plugin name="gazebo_ros_control" filename="libgazebo_ros_control.so"/>
+    </gazebo>
+
+For transmissions and actuators we are going to create a new file, :code:`panda.control.xacro`. As before, I will provide the full content now:
+
+.. code-block:: xml
+
+    <!-- panda.control.xacro -->
+    <?xml version="1.0"?>
+    <robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+        <xacro:macro name="panda_control" params="arm_id">
+            <xacro:macro name="arm_control" params="transmission joint motor">
+                <transmission name="${transmission}">
+                    <type>transmission_interface/SimpleTransmission</type>
+                    <joint name="${joint}">
+                        <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
+                    </joint>
+                    <actuator name="${motor}">
+                        <hardwareInterface>hardware_interface/EffortJointInterface</hardwareInterface>
+                        <mechanicalReduction>1</mechanicalReduction>
+                    </actuator>
+                </transmission>
+            </xacro:macro>
+            <xacro:arm_control transmission="${arm_id}_tran_1" joint="${arm_id}_joint1" motor="${arm_id}_motor_1"/>
+            <xacro:arm_control transmission="${arm_id}_tran_2" joint="${arm_id}_joint2" motor="${arm_id}_motor_2"/>
+            <xacro:arm_control transmission="${arm_id}_tran_3" joint="${arm_id}_joint3" motor="${arm_id}_motor_3"/>
+            <xacro:arm_control transmission="${arm_id}_tran_4" joint="${arm_id}_joint4" motor="${arm_id}_motor_4"/>
+            <xacro:arm_control transmission="${arm_id}_tran_5" joint="${arm_id}_joint5" motor="${arm_id}_motor_5"/>
+            <xacro:arm_control transmission="${arm_id}_tran_6" joint="${arm_id}_joint6" motor="${arm_id}_motor_6"/>
+            <xacro:arm_control transmission="${arm_id}_tran_7" joint="${arm_id}_joint7" motor="${arm_id}_motor_7"/>
+            <xacro:arm_control transmission="${arm_id}_leftfinger" joint="${arm_id}_finger_joint1" motor="${arm_id}_finger_joint1"/>
+            <xacro:arm_control transmission="${arm_id}_rightfinger" joint="${arm_id}_finger_joint2" motor="${arm_id}_finger_joint2"/>
+        </xacro:macro>
+    </robot>
+
+Similarly add the following line to the end of :code:`franka_description/robots/panda_arm_hand_urdf.xacro` file:
```

Comment:
Again, before `</robot>`, correct? Please specify.

Reply:
I really think _end of_ means just before `</robot>` in this context.